operate/client/src/App/BatchOperations/index.tsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {useMemo, useEffect, useState} from 'react';
+import {useSearchParams, Link} from 'react-router-dom';
+import {Breadcrumb, BreadcrumbItem, Pagination} from '@carbon/react';
+import {Locations} from 'modules/Routes';
+import {PAGE_TITLE} from 'modules/constants';
+import {formatDate} from 'modules/utils/date';
+import {Forbidden} from 'modules/components/Forbidden';
+import {HTTP_STATUS_FORBIDDEN} from 'modules/constants/statusCode';
+import {SortableTable} from 'modules/components/SortableTable';
+import {VisuallyHiddenH1} from 'modules/components/VisuallyHiddenH1';
+import {usePaginatedBatchOperations} from 'modules/queries/batch-operations/usePaginatedBatchOperations';
+import {
+  PageContainer,
+  PageHeader,
+  PageWrapper,
+  PanelHeader,
+  TableContainer,
+} from './styled';
+import {parseBatchOperationsSearchSort} from 'modules/utils/filter/v2/batchOperationsSearchSort';
+import {BatchItemsCount} from './BatchItemsCount';
+import {BatchStateIndicator} from './BatchStateIndicator';
+
+const MAX_OPERATIONS_PER_REQUEST = 20;
+
+const BatchOperations: React.FC = () => {
+  const [params] = useSearchParams();
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(MAX_OPERATIONS_PER_REQUEST);
+  const sort = parseBatchOperationsSearchSort(params);
+
+  useEffect(() => {
+    document.title = PAGE_TITLE.BATCH_OPERATIONS;
+  }, []);
```

Comment:
[I think we can use the title tag instead](https://18.react.dev/reference/react-dom/components/title)